src/tls.cpp

```diff
@@ -170,6 +170,11 @@ TLSInfo_GetTidOffset(void)
   }
 #endif
 
+#ifdef __riscv
+  offset = 208;
```

Comment:
@aayushi363 , Please change this from `200` to `192`.  I think we agreed elsewhere that this should be 192, as pointed out by @karya0  (and confirmed by you).

Reply:
@gc00 I don't know why you are still seeing #include I have already changed it to desired formatting.